pkg/logs/agent.go

```diff
@@ -34,12 +36,15 @@ type Agent struct {
 	destinationsCtx  *sender.DestinationsContext
 	pipelineProvider pipeline.Provider
 	inputs           []restart.Restartable
+	health           *health.Handle
 }
 
 // NewAgent returns a new Agent
 func NewAgent(sources *config.LogSources, services *service.Services, endpoints *config.Endpoints) *Agent {
+	h := health.Register("logs-agent")
```

Comment:
nits: not a huge fan of `h`, you should be able to reuse `health` I think

Reply:
changed